code/modules/mob/living/simple_animal/friendly/drone/extra_drone_types.dm

```diff
@@ -54,6 +54,18 @@
 	..()
 	desc += " This drone appears to have a complex holoprojector built on its 'head'."
 
+/mob/living/simple_animal/drone/snowflake/bardrone
+	name = "Bardrone"
+	desc = "A barkeeping drone, an indestructible robot built to tend bars."
+	has_number = FALSE
+	seeStatic = FALSE
+	laws = "1. Serve drinks.\n\
+		2. Talk to patrons.\n\
+		3. Don't get messed up in their affairs."
+	languages = ALL
+	status_flags = GODMODE // Please don't punch the barkeeper
+	unique_name = TRUE
```

Comment:
Needs to be FALSE for it to not generate a number, you gotta put unique_name = TRUE on the base drone